assets/styles/main.css

```diff
@@ -68,7 +66,6 @@ object {
     grid-template-columns: repeat(2, 1fr);
     grid-template-rows: repeat(4, 1fr);
     height: calc(72px * 4 + 5);
-    transform: translate(-50%, -5%);
   }
   z-index: 1;
 }
```

Comment:
The changes to `.sidebar` broke the mobile view of the page.


<img width="774" alt="Screenshot 2023-06-10 at 1 22 52 PM" src="https://github.com/cse110-sp23-group1/Origami-Fortune-Teller/assets/72187062/0ba3c276-1b45-43f3-b869-9cc2296c8b8a">